packages/ui/package.json

```
@@ -0,0 +1,51 @@
+{
+  "name": "@dzcode.io/ui",
+  "description": "dzcode ui library",
+  "version": "1.0.0",
+  "author": {
+    "email": "sohaib.code@gmail.com",
+    "name": "Souhaib Benbouzid",
+    "url": "https://sohaibbenbouzid.com/"
+  },
+  "dependencies": {
+    "@emotion/react": "^11.7.0",
+    "@emotion/styled": "^11.6.0",
+    "@fontsource/roboto": "^4.5.1",
+    "@mui/icons-material": "^5.2.0",
+    "@mui/material": "^5.2.2",
+    "@types/uuid": "^8.3.3",
+    "react": "^17.0.2",
+    "stylis-plugin-rtl": "^1"
+  },
+  "devDependencies": {
+    "@dzcode.io/tooling": "*",
+    "@testing-library/jest-dom": "^5.16.0",
+    "@testing-library/react": "^12.1.2",
+    "@testing-library/user-event": "^13.5.0",
+    "uuid": "^8.3.2"
+  },
+  "license": "MIT",
+  "lint-staged": {
+    "*.*": [
+      "yarn lint:eslint --fix",
+      "yarn lint:prettier --write"
+    ]
+  },
+  "main": "dist",
+  "private": true,
+  "scripts": {
+    "build": "yarn lerna run build:alone --scope=@dzcode.io/ui --include-dependencies --stream",
+    "build:alone": "tsc",
+    "build:alone:watch": "tsc --watch --preserveWatchOutput",
+    "build:watch": "yarn build & yarn lerna run build:alone:watch --scope=@dzcode.io/ui --include-dependencies --parallel",
+    "clean": "yarn lerna run clean:alone --scope=@dzcode.io/ui --include-dependencies --stream",
+    "clean:alone": "rimraf dist coverage",
+    "lint": "yarn build && yarn lint:alone",
+    "lint:alone": "yarn lint:eslint . && yarn lint:prettier --check .",
+    "lint:eslint": "eslint --config ../tooling/.eslintrc.json --ignore-path ../tooling/.eslintignore --report-unused-disable-directives",
```

We recently added these two scripts, you gotta add them too

```suggestion
    "lint:eslint": "eslint --config ../tooling/.eslintrc.json --ignore-path ../tooling/.eslintignore --report-unused-disable-directives",
    "lint:fix": "yarn build && yarn lint:fix:alone",
    "lint:fix:alone": "yarn lint:eslint --fix . && yarn lint:prettier --write .",
```

---

0e363a7bbd6186b1a450ba83bd994e4459b18b24